applications/FemToDemApplication/python_scripts/MainSolidFEM.py

```diff
@@ -7,7 +7,6 @@
 
 # Import kratos core and applications
 import KratosMultiphysics
-import KratosMultiphysics.SolidMechanicsApplication as KratosSolid
 
 class Solution(object):

```

Comment:
```suggestion
class Solution:

```

this is no longer needed since we dropped python2 support